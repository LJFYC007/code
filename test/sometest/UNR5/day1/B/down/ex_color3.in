100
17 44
86 12
64 29
44 47
83 40
10 59
100 39
57 85
54 32
26 44
44 11
44 58
44 6
7 54
53 21
44 69
42 35
92 93
86 67
42 44
74 60
14 97
15 57
31 57
4 100
44 84
25 69
11 89
16 44
44 34
23 27
17 78
95 21
20 27
79 44
8 69
3 14
45 14
44 27
33 41
71 58
12 44
27 98
77 10
5 44
84 90
14 18
13 89
88 58
54 9
60 36
96 58
66 2
28 79
49 43
44 77
21 44
55 65
44 100
60 19
80 44
44 40
26 29
62 44
68 80
78 82
87 92
33 44
52 33
21 30
73 100
57 44
27 51
100 61
44 38
34 48
46 44
91 48
37 11
72 24
1 92
70 24
33 99
40 43
44 14
60 44
44 32
76 6
56 90
81 69
50 80
5 65
66 75
70 44
44 93
63 80
2 44
33 94
21 22
