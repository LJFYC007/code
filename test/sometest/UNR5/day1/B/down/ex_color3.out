18 356 7521 166952 3854148 91900925 256301193 681149376 229333151 133460718 906089768 277109652 600350404 134339409 469587823 818213940 24425263 816391366 161082659 425387435 84661786 625590634 463427282 101437535 970383425 699898746 718359925 134920222 238535072 10685318 219723196 323407089 589110419 271828731 162371409 776891531 595379486 601080068 835261267 947805971 480105239 490001928 654985468 107207527 657939032 582513880 146360301 287638623 219545573 53298328 381045524 925993 955145947 344349991 334559719 54815125 257345752 678544218 802010751 929749508 814144990 808721398 982409083 657091857 880403789 30459160 324676821 488650185 882325742 907408945 660265921 262535525 265778554 397114533 842005619 405007912 992763039 294186030 368905455 891665906 659508438 782587023 388062 806678733 53569239 856941206 220210378 606982089 100724661 838949579 981965655 375536192 490771390 359723624 429415258 702503105 998216651 393237408 71154503 71154503
